frameworks/kafka/setup-helper/setup_helper.go

```diff
@@ -26,6 +26,7 @@ const (
 	authorizationEnvvar   = "SECURITY_AUTHORIZATION_ENABLED"
 	superUsersEnvvar      = "SECURITY_AUTHORIZATION_SUPER_USERS"
 	brokerCountEnvvar     = "BROKER_COUNT"
+	advertiseHostIPEnvvar = "KAFKA_ADVERTISE_HOST"
```

Comment:
Does this meant hat we no longer use the value set through config.json?

Reply:
No, I'm obeying the envvar that the config.json value sets.

Reply:
https://github.com/mesosphere/dcos-commons/blob/master/frameworks/kafka/universe/marathon.json.mustache#L116